spec/features/checkout_feature_spec.rb

```diff
@@ -14,7 +14,7 @@
   after(:all) { DatabaseCleaner.clean_with(:truncation) }
 
   context "when not logged in" do
-    it "redirects to login page if not logged in", js: true do
+    it "redirects to login page if not logged in", :redirect_login, js: true do
```

Comment:
Thanks, @andela-bmakinwa. I will change the description. However, the description was made by my predecessors.